### PR TITLE
Vet clinic Database project module four.

### PR DESCRIPTION
**Pull request by juaryimami**
### In this project,
---
- I use a relational database to create the initial data structure for a vet clinic.
-  I also create a table to store animals' information, insert some data into it, and query it.
- I have attached the screenshot of all of my sql query as shown below.

Screenshot from 2023-01-16 18-15-11
Screenshot from 2023-01-16 18-17-14
Screenshot from 2023-01-16 22-00-11
Screenshot from 2023-01-17 08-42-51
Screenshot from 2023-01-17 08-47-40
Screenshot from 2023-01-17 08-57-58

Screenshot from 2023-01-16 22-08-42
Screenshot from 2023-01-16 22-16-01
Screenshot from 2023-01-16 22-20-30
---
#### Thank you for reviewing my project!!!!.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals;
+SELECT * from animals where name like '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-01-01';
```
Comment (BenMukebo) [inline queries.sql]: - [ ] For this requirement, ` list the name of all animals born between 2016 and 2019.` Here you should list the name of all animals starting from the first day of the first month of 2016 up to the last day of the last month of 2019, hope this makes sense 🤔 Your code might look like this:

```sql
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
# 0r 
SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;
```


- Please correct this 👍🏻

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT name from animals where escape_attempts <3;
```
Comment (BenMukebo) [inline queries.sql]: - [ ] For this one, `list the name of all animals that are neutered and have less than 3 escape attempts.` You are missing adding conditions for the `neutered` case. kindly add this. Your code might look like this:

```sql
SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';
```

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+SELECT date_of_birth from animals where name = 'Agumon' or  name = 'Pikachu';
+SELECT name, escape_attempts from animals where weight_kg >10;
```
Comment (BenMukebo) [inline queries.sql]: - [ ] Let's be more consistent 🥲 `list name and escape attempts of animals that weigh more than 10.5kg` as you can see, it is 10.5 and not 10. Please correct this 👍🏻 

```sql
SELECT name, escape_attempts FROM animals WHERE weight_kg > '10.5';
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @juaryimami,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Gitflow was correctly used.  ✔️   
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ]  Please consider updating the screenshots of your queries results to the pull request description. This also helps to ensure that you run the queries with success.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by juaryimami** [comment]
Ok thanks sir!

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @juaryimami  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:
- Made changes as requested by reviewer.
- 
## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by juaryimami** [comment]
Thank you!

**Comment by omarsalem7** [comment]
Hi 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**